pressbooks/pressbooks
# Remove MOBI export and references

This PR removes the MOBI export option and most MOBI related references from Pressbooks. Partial fix for https://github.com/pressbooks/pressbooks/issues/2193 and fix for https://github.com/pressbooks/pressbooks/issues/1859. 

It also makes a small change to the export format menu, moving 
PressbooksXML and Thin Common Cartridge with weblinks to the 'supported formats' column and renaming the EPUB 2.01 export option.

![Screenshot from 2021-07-15 19-38-14](https://user-images.githubusercontent.com/13485451/125883238-5580afd7-9524-4795-a014-86e7366f5d89.png)

**To test:**
1. Turn on ability to distribute files from home page
1. Open a book and create a MOBI export
1. Create an EPUB export and confirm that it does not have validation errors
1. Switch to this branch and open the export menu in any book
1. Check to see that MOBI export no longer appears in the menu
1. Check to see that the previously created MOBI file still appears in the list of previously exported files & can still be downloaded from the book's home page.
1. Confirm that EPUB 2.01 exports do appear in the menu 
1. Create an EPUB export and confirm that it still does not have validation errors
